app/src/main/java/org/mozilla/fenix/library/downloads/DownloadFragmentStore.kt

```diff
@@ -16,7 +16,8 @@ import mozilla.components.lib.state.Store
  * @property size The size in bytes of the download item
  * @property contentType The type of file the download is
  */
-data class DownloadItem(val id: Long, val fileName: String?, val filePath: String, val size: String, val contentType: String?)
+data class DownloadItem
+    (val id: Long, val fileName: String?, val filePath: String, val size: String, val contentType: String?)
```

Comment:
nit: we usually format these differently: https://github.com/mozilla-mobile/fenix/blob/52c2fdb310f5b86ce6484821b4901f85f15ee858/app/src/main/java/org/mozilla/fenix/home/HomeFragmentStore.kt#L25